### DISCUSSION TITLE
httpx requirement

### DISCUSSION THREAD

**Discussion by jubalh** [mkb79/audible-cli#74] [Q&A] [answered]
Hi!
Do you really require exactly httpx==0.20.*? Newer versions like httpx 0.22 won't work?

**Comment by mkb79** [comment]
Hi!
I've take a look on the changelog of httpx. Theoretically 0.22 should work. I can't test this out yet. Pythonista for iOS doesn't support httpx>=0.21. Maybe you can check this and report your result to me. Or you have to wait until I can test this with my Windows machine.

**Comment by mkb79** [comment, marked as answer]
httpx 0.22.0 seams to work with my packages. So I allowed httpx 0.20-> 0.22 now!

**Reply by jubalh** [reply]
Thanks!
